.env.template

```
@@ -1,115 +1,124 @@
-isDevelopmentMode=enabled
-ENV=development
+
+# App & Environment
+APP_NAME=potpie-ai
+APP_VERSION=0.1.0
+ENV=development            # development | staging | production
+isDevelopmentMode=enabled   
```

dotenv-linter (4.0.0)

[warning] [LowercaseKey] The isDevelopmentMode key should be in uppercase

(LowercaseKey)